test/server/servers/SockJSServer.test.js

```diff
@@ -10,7 +10,7 @@ describe('SockJSServer', () => {
   let socketServer;
   let listeningApp;
 
-  beforeAll((done) => {
+  beforeEach((done) => {
```

Comment:
These can stay as `beforeAll` and `afterAll`. It is reusing the same server for all tests. Switching to `beforeEach/afterEach` would not change anything because there is only that one `describe` call on the same level as these calls.

Reply:
* Reason for using `beforeEach`:
As my test needs to test null scenario and we are mutating the same environment, it shows
![image](https://user-images.githubusercontent.com/5212215/63313663-0da2f600-c2ba-11e9-9d0d-f0a2e8ec72f8.png)

* Reason for the `timeout`
It won't pass in the build machine only for Windows node-8 and Windows node-6.
![image](https://user-images.githubusercontent.com/5212215/63313860-bcdfcd00-c2ba-11e9-8bb0-89565b9bca2e.png)
See https://dev.azure.com/webpack/webpack-dev-server/_build/results?buildId=5531

Reply:
Yes that makes sense. My understanding of `beforeEach/afterEach` in describe blocks was incorrect.